src/components/input-style/input-style.html

```diff
@@ -15,38 +15,41 @@
      @mousedown="onMousedown"
      @mouseup="onMouseup"
      ref="root"
-     :style="{ width: width,  marginTop: labelOffset }">
-    <label class="m-input-style__label"
-           ref="label"
-           :for="labelFor"
-           v-if="hasLabel">
-        <span class="m-input-style__text">
-            <span>{{ label }}</span>&nbsp;<sup class="m-input-style__required-marker"
-                 v-if="requiredMarker">*</sup>
-        </span>
-    </label>
-    <div class="m-input-style__body"
-         :style="{ top: bodyOffset }"
-         ref="body">
-        <div class="m-input-style__body__left">
-            <slot></slot>
-            <m-i18n k="m-input-style:empty-value"
-                    class="m-input-style__body__transparent-text"
-                    aria-hidden="true"
-                    v-if="!hasDefaultSlot"></m-i18n>
-            <div class="m-input-style__body__adjust-width-auto"
-                 ref="adjustWidthAuto"
-                 aria-hidden="true"
-                 v-if="hasAdjustWidthAutoSlot">
-                <slot name="adjust-width-auto"></slot>
+     :style="{ width: width }">
+    <div class="m-input-style__content"
+         :style="{ marginTop: labelOffset }">
+        <label v-if="hasLabel"
+               :for="labelFor"
```

Comment:
Dans Brio, on est rendus avec un ordre pour mettre les attributs. Je ne sais pas trop si Modul suit les mêmes, mais ça pourrait être pas pire.

1. Attributs de contrôles (v-if, v-else, v-else-if, v-show, v-for avec :key)
2. id
3. class
4. :class
5. v-model
6. toutes les props regroupées ensemble (:xyz ou v-bind)
7. tous les événements regroupés ensemble (@xyz ou v-on)
8. autre directives vue : v-xxx
9. autres attributs dans un ordre logique

https://wiki.dti.ulaval.ca/pages/viewpage.action?pageId=75528135

Reply:
Les `refs` on les positionne où?

Reply:
Elles rentrent dans le 9.

Reply:
Dans ce cas-ci, ce n'est pas un `v-for`, mais bien l'Attribut HTML `for`

Reply:
J'ai juste mal tapé. Faudrait les inverser pareil. La classe est censée être en 2e.

Reply:
Un `for` HTML est considéré comme un attribut de contrôle?

Reply:
C'est pas le for le problème, c'est le class qui est trop bas.